00_basics/py_dimensions.ipynb

```diff
@@ -0,0 +1,740 @@
+{
```

Comment:
<p>to the large the number of third-party libraries</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/capital-G/musikinformatik-sose2021/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>